lib/modules/manager/github-actions/extract.ts

```diff
@@ -198,34 +172,31 @@ function extractWithYAMLParser(
   logger.trace('github-actions.extractWithYAMLParser()');
   const deps: PackageDependency[] = [];
 
-  let pkg: Workflow;
-  try {
-    // TODO: use schema (#9610)
-    pkg = parseSingleYaml(content);
-  } catch (err) {
-    logger.debug(
-      { packageFile, err },
-      'Failed to parse GitHub Actions Workflow YAML',
-    );
-    return [];
-  }
+  const jobs = WorkflowJobsSchema.parse(content);
```

Comment:
please add some error logging, so we know if parsing fails (probably of some github changes)